content/developer/writing_widgets/widget-definition.md

```diff
@@ -141,6 +141,10 @@ fetchUrl: '[manager]/executions?is_system_workflow=false[params]'
      Please note that this can be used both to selectively pick pagination parameter as well as custom parameters (see [fetchParams function](#fetchparams-widget-toolbox)).
       This mode is **exclusive** - parameters not specified explicitly are skipped.
       When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include all pagination parameters (`_size`, `_offset`, `_sort`) instead of specifying explicitly  each of the three.
+      It is possible to specify an additional search parameter (`_search`) to filter rows, 
+      e.g. `fetchUrl: '[manager]/agents?[params:_offset,_size,_sort,_search]'`.
+      Hint: `DataTable` component provides the `searchable` property that should be set as `true` to provide the `_search` parameter.
```

Comment:
Please check "How to use sorting/pagination/searching DataTable features with external APIs?" in https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/master/content/developer/writing_widgets/faq.md . Maybe we can just reference it here and not add specific description about the usage. 

Btw, there's one formatting issue in the referenced faq.md file, so you can update at once.

Reply:
formatting fixed:
https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/RD-1323-added-search-parameter-to-docs/content/developer/writing_widgets/faq.md